blender/arm/material/make_depth.py

```diff
@@ -188,7 +233,14 @@ def make(context_id, rpasses, shadowmap=False):
                 vert.add_out('vec3 vcolor')
                 vert.write('vcolor = col.rgb;')
 
-    if parse_opacity:
+    if shadowmap_transparent:
+        frag.add_out('vec4 fragColor')
+        frag.write('float depth = wposition.z;')
+        frag.write('vec3 color = basecol;')
+        frag.write('color *= 1.0 - opacity;')
```

Comment:
According to juanos, opacity should be modified by transmittance: opacity = 1.0 - transmittance.